dll/win32/shell32/lang/he-IL.rc

```diff
@@ -341,15 +343,15 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
     GROUPBOX "בדיקת שגיאות", -1, 5, 5, 230, 60
     ICON IDI_SHELL_TOOLS_CHKDSK, IDC_STATIC, 13, 25, 21, 20
-    LTEXT "This option will check the volume for\nerrors.", -1, 40, 25, 160, 20
+    LTEXT "אפשרות זו תבדוק אם ישנן שגיאות בכונן.", -1, 40, 25, 160, 20
```

Comment:
This change seems RTL'ed broken.

Reply:
this Change was made by @peterooch he'll review this PR soon.

Reply:
i'll ask Peter about it.

Reply:
My name is not Peter...

Reply:
Baruch*

Reply:
@binarymaster what do you mean by "this change seems RTL'ed broken?" in what way would it be broken?

Reply:
> what do you mean by "this change seems RTL'ed broken?"

The screenshot attached describes it.

![image](https://user-images.githubusercontent.com/578406/62951773-370fdd80-bdf3-11e9-9c20-2ecf54feac2d.png)

Reply:
Wouldn't this be the same issue as before? 
Since once again. this does appear correctly in the file itself when viewed in notepad. 
and it should work just fine infact CL marked it as Okay.